examples/appveyor-minimal.yml

```diff
@@ -1,14 +1,20 @@
-image:
-  - Ubuntu
-  - Visual Studio 2015
-
-build_script:
-  # windows
-  - cmd: pip3 install cibuildwheel==1.1.0
-  - cmd: cibuildwheel --output-dir wheelhouse
-  # linux
-  - sh: "${HOME}/.localpython3.7.4/bin/python3 -m pip install cibuildwheel==1.1.0"
-  - sh: "${HOME}/.localpython3.7.4/bin/python3 -m cibuildwheel --output-dir wheelhouse"
+environment:
+  matrix:
+    - APPVEYOR_BUILD_WORKER_IMAGE: Ubuntu
+      APPVEYOR_JOB_NAME: "python37-x64-ubuntu"
+    - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
+      APPVEYOR_JOB_NAME: "python37-x64-vs2015"
+    - APPVEYOR_BUILD_WORKER_IMAGE: macos-mojave
+      APPVEYOR_JOB_NAME: "python37-x64-macos-mojave"
+
+stack: python 3.7
+
+init:
+- cmd: set PATH=C:\Python37;C:\Python37\Scripts;%PATH%
+
+install: python -m pip install cibuildwheel==1.1.0
```

Comment:
If this is just `python`, will it run with Python3 on macOS? Normally `python` means Python 2 on macOS, unless Appveyor is doing something clever. I guess if it builds, that answers our question!

Reply:
We're using the python stack feature of AppVeyor which setups a virtualenv (macOS/Linux only) so we're using python 3.7

Reply:
PS: it builds, https://ci.appveyor.com/project/joerick/cibuildwheel/builds/31310546
I've rebased the branch on latest master, that's why the status is pending.

Reply:
Cool. Thanks!